wp-content/themes/csisnuclearnetwork/functions.php

```diff
@@ -233,6 +235,11 @@ function nuclearnetwork_register_scripts() {
 	wp_enqueue_script( 'nuclearnetwork-js-bundle', get_template_directory_uri() . '/assets/js/bundle.min.js', array(), $theme_version, true );
 	wp_script_add_data( 'nuclearnetwork-js-bundle', 'defer', true );
 
+	if ( is_front_page() ) {
```

Comment:
This is fine for now, but if you end up needing to add more JS to the home page, you  might consider making a `home.js` and importing the `carousel.js` file from there.